### PR TITLE
Update data retrieval offer

### PR DESCRIPTION
**Pull request by tsolakoua**
Signed-off-by: tsolakou <tsolakou.a@gmail.com>

Update after @mcuadros request. If it's approved, I will update Lever

### CODE REVIEW

**Comment by tsolakoua** [comment]
@ajnavarro please check if the change I made makes sense. If no feel free to suggest one that fits better.

File: talent/job-descriptions/offers/data-retrieval.md [hunk 1]
```diff
@@ -9,15 +9,15 @@ Software Engineer in the Data Retrieval team
 
 ### [Engineering at source{d}](../engineering-section.md)
 
-### Role 
+### Role
 
-The Data Retrieval team is developing source{d}'s high-level Engine for running scalable data retrieval pipelines that process and manipulate any number of code repositories for source code analysis. Written mostly in Scala, it aims to be robust, friendly and flexible: it is built on top of Apache Spark, accessible both via Scala and Python Spark APIs, and capable of running on large-scale distributed clusters over petabytes of data.
+The Data Retrieval team is developing source{d}'s high-level Engine for running scalable data retrieval pipelines that process and manipulate any number of code repositories for source code analysis. Written mostly in Go, it aims to be robust, friendly and flexible: it is built on top of Apache Spark, accessible both via Scala and Python Spark APIs, and capable of running on large-scale distributed clusters over petabytes of data.
```
Comment (ajnavarro) [inline talent/job-descriptions/offers/data-retrieval.md]: high-level Engine -> high-level code analysis applications

Comment (ajnavarro) [inline talent/job-descriptions/offers/data-retrieval.md]: remove `it is built on top of Apache Spark`

Comment (ajnavarro) [inline talent/job-descriptions/offers/data-retrieval.md]: I think we should rethink this paragraph.

Comment (tsolakoua) [reply talent/job-descriptions/offers/data-retrieval.md]: @ajnavarro I made some changes based on your comments, if you still think it needs to be improved we can do it together.

**Pull request merged**